db/migrate/20230222123036_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] As requested in the comment above, the reference for the user in the likes table, should be author_id and not user_id. Kindly refer to the first comment above on how to implement this.

![image](https://user-images.githubusercontent.com/86318284/220627792-6f103c1f-b36f-48e0-9dcc-9049ff1e63ef.png)